src/Microsoft.AspNetCore.Mvc.Core/Internal/IApiBehaviorMetadata.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
```

Comment:
This isn't really the right place for this interface, but we don't have another good **existing** one.

Reply:
What exactly is this interface? I read the doc comment but it isn't clear to me why it's needed. What else would implement it? Or is it to avoid coupling to the `ApiControllerAttribute`?

Reply:
Yep. The last one. 

For instance this attribute is only supported on classes, because that's how we image you'll want to use it. But there's nothing wrong with you implementing your own marker.

It short, there's value in separating policy from specification

Reply:
Got it, makes sense.